rtos/mbed_lib.json

```diff
@@ -1,6 +1,198 @@
 {
     "name": "rtos",
     "config": {
-        "present": 1
+        "present": 1,
+
+        "taskcnt": {
+            "help": [
+                "Number of concurrent running user threads <1-250>",
```

Comment:
We did consider YAML. In fact, the original design of the config system assumed YAML. We switched to JSON for reasons that I'd rather not get into, but the fact is that we use JSON and that's that. I don't know if there's a good way to do multiline strings in JSON. However, it is fairly easy to modify the implementation of the config system to accept both lists and simple strings for the "help" field.